### ISSUE TITLE
"exts" does not work ?

### ISSUE THREAD

**Issue by kght6123** [nhnent/toast-ui.vue-editor#7]
# question

## Version
1.0.3

## Test Environment
Safari 12.0, macOS Mojave 10.14

## Current Behavior
I am looking for an option to use exts.

```vue
<template>
  <div id="app">
    <editor v-model="editorText" :options="editorOptions" />
  </div>
</template>

<script>
import { Editor } from '@toast-ui/vue-editor'

export default {
  components: {
    'editor': Editor,
  },
  data() {
      return {
          editorText: '# CCC',
          editorOptions: {
            exts: ['scrollSync', 'colorSyntax', 'uml', 'chart', 'mark', 'table'],
          }
      }
  },
}
</script>
```

Thank you.

**Comment by sohee-lee7** [comment]
When use `exts`, should import exts files like this.
```js
import 'tui-editor/dist/tui-editor-extScrollSync'
import 'tui-editor/dist/tui-editor-extColorSyntax'
import 'tui-editor/dist/tui-editor-extUML'
import 'tui-editor/dist/tui-editor-extChart'
import 'tui-editor/dist/tui-editor-extTable'
```

**Comment by kght6123** [comment]
Thank you for answering.

I added import, but it did not solve it.

Below is detailed code.

Are there other possible causes?

```vue
<template>
  <div id="app">
    <editor v-model="editorText" :options="editorOptions" previewStyle="vertical" height="900px" />
  </div>
</template>

<script>
import 'tui-editor/dist/tui-editor-extScrollSync'
import 'tui-editor/dist/tui-editor-extColorSyntax'
import 'tui-editor/dist/tui-editor-extUML'
import 'tui-editor/dist/tui-editor-extChart'
import 'tui-editor/dist/tui-editor-extTable'

import { Editor } from '@toast-ui/vue-editor'

export default {
  components: {
    'editor': Editor,
  },
  data() {
      return {
          editorText: '# CCC'+'\n'+
''+'\n'+
'| @cols=2:merged |'+'\n'+
'| --- | --- |'+'\n'+
'| table | table |'+'\n'+
''+'\n'+
'```chart'+'\n'+
',category1,category2'+'\n'+
'Jan,21,23'+'\n'+
'Feb,31,17'+'\n'+
''+'\n'+
'type: column'+'\n'+
'title: Monthly Revenue'+'\n'+
'x.title: Amount'+'\n'+
'y.title: Month'+'\n'+
'y.min: 1'+'\n'+
'y.max: 40'+'\n'+
'y.suffix: $'+'\n'+
'```'+'\n'+
''+'\n'+
'```uml'+'\n'+
'partition Conductor {'+'\n'+
'  (*) --> "Climbs on Platform"'+'\n'+
'  --> === S1 ==='+'\n'+
'  --> Bows'+'\n'+
'}'+'\n'+
''+'\n'+
'partition Audience #LightSkyBlue {'+'\n'+
'  === S1 === --> Applauds'+'\n'+
'}'+'\n'+
''+'\n'+
'partition Conductor {'+'\n'+
'  Bows --> === S2 ==='+'\n'+
'  --> WavesArmes'+'\n'+
'  Applauds --> === S2 ==='+'\n'+
'}'+'\n'+
''+'\n'+
'partition Orchestra #CCCCEE {'+'\n'+
'  WavesArmes --> Introduction'+'\n'+
'  --> "Play music"'+'\n'+
'}'+'\n'+
'```',
          editorOptions: {
            exts: ['scrollSync', 'colorSyntax', 'uml', 'chart', 'mark', 'table'],
          }
      }
  },
}
</script>
```

<img width="1071" alt="2019-01-30 14 49 50" src="https://user-images.githubusercontent.com/24556516/51961137-17404a00-249f-11e9-8d76-4a7a9d1f1589.png">

I think that it is necessary to import in `toast-ui.vue-editor/src/Editor.vue` and `Viewer.vue`.

Thank you.

**Comment by sohee-lee7** [comment]
When I try your code, exts work well. Please give me more information about bundler or other settings.

```js
<template>
    <div id="app">
        <editor
        v-model="editorText"
        :options="editorOptions"
        height="900px"
        previewStyle="vertical"
        />
    </div>
</template>
<script>
import 'tui-editor/dist/tui-editor.css';
import 'tui-editor/dist/tui-editor-contents.css';
import 'codemirror/lib/codemirror.css';
import 'tui-editor/dist/tui-editor-extScrollSync'
import 'tui-editor/dist/tui-editor-extColorSyntax'
import 'tui-editor/dist/tui-editor-extUML'
import 'tui-editor/dist/tui-editor-extChart'
import 'tui-editor/dist/tui-editor-extTable'
import { Editor } from '@toast-ui/vue-editor'

export default {
    name: 'App',
    components: {
        'editor': Editor
    },
    data() {
        return {
            editorText: '# CCC'+'\n'+
                        ''+'\n'+
                        '| @cols=2:merged |'+'\n'+
                        '| --- | --- |'+'\n'+
                        '| table | table |'+'\n'+
                        ''+'\n'+
                        '```chart'+'\n'+
                        ',category1,category2'+'\n'+
                        'Jan,21,23'+'\n'+
                        'Feb,31,17'+'\n'+
                        ''+'\n'+
                        'type: column'+'\n'+
                        'title: Monthly Revenue'+'\n'+
                        'x.title: Amount'+'\n'+
                        'y.title: Month'+'\n'+
                        'y.min: 1'+'\n'+
                        'y.max: 40'+'\n'+
                        'y.suffix: $'+'\n'+
                        '```'+'\n'+
                        ''+'\n'+
                        '```uml'+'\n'+
                        'partition Conductor {'+'\n'+
                        '  (*) --> "Climbs on Platform"'+'\n'+
                        '  --> === S1 ==='+'\n'+
                        '  --> Bows'+'\n'+
                        '}'+'\n'+
                        ''+'\n'+
                        'partition Audience #LightSkyBlue {'+'\n'+
                        '  === S1 === --> Applauds'+'\n'+
                        '}'+'\n'+
                        ''+'\n'+
                        'partition Conductor {'+'\n'+
                        '  Bows --> === S2 ==='+'\n'+
                        '  --> WavesArmes'+'\n'+
                        '  Applauds --> === S2 ==='+'\n'+
                        '}'+'\n'+
                        ''+'\n'+
                        'partition Orchestra #CCCCEE {'+'\n'+
                        '  WavesArmes --> Introduction'+'\n'+
                        '  --> "Play music"'+'\n'+
                        '}'+'\n'+
                        '```',
            editorOptions: {
                exts: ['scrollSync', 'colorSyntax', 'uml', 'chart', 'mark', 'table']
            }
        };
    }
};
</script>
```
![image](https://user-images.githubusercontent.com/42666492/52025955-ae61dc00-2549-11e9-8349-68610174226a.png)

**Comment by kght6123** [comment]
I tried but exts does not work.
It seems there is a problem with my environment and other modules.
I'm sorry.

Thank you very much.

**Comment by rodumani** [comment]
I had the same problem that none of extensions worked and I found the reason.

I was using `tui-editor` version 1.2.9 and the problem has resolved after I upgraded it to 1.3.3 

If anyone who has the same problem, try to upgrade `tui-editor`.
